qutebrowser/config/configdata.yml

```diff
@@ -1012,6 +1012,31 @@ input.spatial_navigation:
     Right key, heuristics determine whether there is an element he might be
     trying to reach towards the right and which element he probably wants.
 
+# TODO set to false before merging!
+input.blur_on_load.enabled:
```

Comment:
I'm still very conflicted about these setting structure/names, any suggestions?

Reply:
I don't the think the `.enabled` is needed for the first `Bool`.
Instead, I would start the descripton with "Enable" to match other `Bool` options.
Regarding the other `Bool`, I suggest `.leave` instead of `_leave` to better match the `Int` `.deay`.

Reply:
Without the `.enabled`, `input.blur_on_load.delay` would shadow `input.blur_on_load` which isn't allowed.